src/main/java/uk/gov/hmcts/darts/audio/service/impl/AudioTransformationServiceImpl.java

```diff
@@ -207,10 +208,9 @@ private void processAudioRequest(Integer requestId) {
                 mediaRequestEntity
             );
 
-            boolean hasAllMediaBeenCopiedFromInboundStorage = checkAllMediaExistsInTwoStorageLocations(filteredMediaEntities,
-                                                                                          ExternalLocationTypeEnum.INBOUND,
-                                                                                          ExternalLocationTypeEnum.UNSTRUCTURED);
-            if (!hasAllMediaBeenCopiedFromInboundStorage) {
+            boolean hasNotAllMediaBeenCopiedFromInboundStorage = eodService.hasNotAllMediaBeenCopiedFromInboundStorageForAtsProcessing(filteredMediaEntities);
```

Comment:
This might be better to have the logic swapped, so that the boolean is called `hasAllMediaBeenCopiedFromInboundStorage`

Reply:
done, changed the variable and service method